src/store/page-load/saga.test.ts

```diff
@@ -0,0 +1,106 @@
+import { expectSaga } from 'redux-saga-test-plan';
+import { saga } from './saga';
+import { getHistory } from '../../lib/browser';
+import { rootReducer } from '../reducer';
+import { getCurrentUserWithChatAccessToken } from '../authentication/saga';
+import { call } from 'redux-saga/effects';
+import { initializePublicLayout } from '../layout/saga';
+
+jest.mock('../../config', () => ({
+  config: {
+    defaultZnsRoute: 'wilder',
+    defaultApp: 'nfts',
+  },
+}));
+
+jest.mock('../../lib/feature-flags', () => ({
+  featureFlags: {
+    allowPublicZOS: false,
+  },
+}));
```

Comment:
I wasn't able to conditionally mock it. Like in a test i wanted to set this to `true`, so i can test if the `initializePublicLayout` function is called, but i wasn't able to "invoke" this from within a test. Seems to work above the `describe('..', { .. })` level only i think 🤔

Reply:
```
const featureFlags = { allowWeb3Registration: false };
jest.mock('../../lib/feature-flags', () => ({
  featureFlags: featureFlags,
}));
```
Then in the tests:
```
featureFlags.allowWeb3Registration = true;
```

Reply:
thanks, works! Added one more test